Makefile

```diff
@@ -11,10 +11,18 @@ migrate_down_sqlite:
 clear_sqlite:
 	sqlite3 ~/.sematic/db.sqlite3 < sematic/db/scripts/clear_all.sql
 
+prepare-pre-commit:
```

Comment:
Maybe let's name this something more generic like "install-dev-deps" or something.

Reply:
changed!